### ISSUE TITLE
[A11y][Calender]: [Missing date range in the title once we click on the year in calender]

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents-react#7115]
### Describe the bug

In the Calendar OR Date Picker component, When we click on the year, it should display the date range in the title

### Reproduction steps

1. Go to https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/inputs-calendar--docs
2. Turn on screen reader
3. Focus on Object status
4. Check the announcement

### Expected Behaviour

Expected Result:- 

It should display as SAP UI5 Calendar component - https://ui5.sap.com/#/entity/sap.ui.unified.Calendar/sample/sap.ui.unified.sample.CalendarSingleDaySelection 

### Screenshots or Videos

<img width="1153" alt="Image" src="https://github.com/user-attachments/assets/08f12c87-8b8d-4949-adb4-03eac389e4d6" />

### UI5 Web Components for React Version

2.7.2

### UI5 Web Components Version

2.7.3

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @VidhiNPatel 

I will close this issue as there is already another one covering this topic. If you have any questions or more information, please address them in the [original issue](https://github.com/SAP/ui5-webcomponents/issues/3200).
